File: TiffinBox/app/src/main/java/com/tiff/tiffinbox/Seller/addCustomers/map/interfaces2/FirebaseDriverListener.java
package com.tiff.tiffinbox.Seller.addCustomers.map.interfaces2;
import com.tiff.tiffinbox.Seller.addCustomers.map.model2.Driver;


public interface FirebaseDriverListener {
    void onDriverAdded(Driver var1);

    void onDriverRemoved(Driver var1);

    void onDriverUpdated(Driver var1);
}

File: TiffinBox/app/src/main/java/com/tiff/tiffinbox/Seller/addCustomers/map/interfaces2/FirebaseDriverListener.kt
package com.tiff.tiffinbox.Seller.addCustomers.map.interfaces2

import com.tiff.tiffinbox.Seller.addCustomers.map.model2.Driver

interface FirebaseDriverListener {
    fun onDriverAdded(var1: Driver?)
    fun onDriverRemoved(var1: Driver?)
    fun onDriverUpdated(var1: Driver?)
}